spring-session-data-gemfire/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply from: JAVA_GRADLE
+apply from: MAVEN_GRADLE
+
+apply plugin: 'spring-io'
+
+description = "Aggregator for Spring Session and Spring Data GemFire"
+
+dependencies {
+    compile project(':spring-session'), "org.springframework.data:spring-data-gemfire:$springDataGemFireVersion"
```

Comment:
Is it time to split the project in -core and modules like -redis, -hazelcast, -gemfire ?
A user of Redis doesn't need Gemfire jars in her application (scope `compile`).
Another way would be to use scope `provided` and then each application should provide the respective dependency.